src/weather.js

```diff
@@ -0,0 +1,20 @@
+
+class Weather {
+  constructor(city) {
+    this.apiKey = '1a47b95fce72843fb086b93f9d54e7b8';
+    this.city = city;
+  }
+
+  async getWeatherInfo() {
+    const response = await fetch(`http://api.openweathermap.org/data/2.5/weather?q=${this.city}&appid=${this.apiKey}`);
+    const responseData = await response.json();
+    return responseData;
+  }
```

Comment:
- **_It should have an error handling for `promises` or `async/await` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)). This will help to catch any error such as when the request responds with an error because the location is not found_**